jenkins/e2e-image/e2e-runner.sh

```diff
@@ -243,8 +243,10 @@ function setup_gci_vars() {
 
 ### Pre Set Up ###
 if running_in_docker; then
-    record_command "${STAGE_PRE}" "download_gcloud" curl -fsSL --retry 3 --keepalive-time 2 -o "${WORKSPACE}/google-cloud-sdk.tar.gz" 'https://dl.google.com/dl/cloudsdk/channels/rapid/google-cloud-sdk.tar.gz'
-    install_google_cloud_sdk_tarball "${WORKSPACE}/google-cloud-sdk.tar.gz" /
+    if [[ -n "${GOOGLE_APPLICATION_CREDENTIALS:-}" ]]; then
```

Comment:
would it be better to condition on `$KUBERNETES_PROVIDER` being `gce|gke|kubemark`?